mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -2,28 +2,55 @@
 template: main.html
 ---
 
-# Load Test with SLOs
+# Your First Experiment
 
-!!! tip "Load test an HTTP Service and validate SLOs" 
-    Use an [Iter8 experiment](concepts.md#what-is-an-iter8-experiment) to load test an HTTP service and validate latency and error-related [service level objectives (SLOs)](../user-guide/topics/slos.md).
+!!! tip "Learn the basics of an Iter8 experiment"
+    Learn the basics of an [Iter8 experiment](concepts.md#what-is-an-iter8-experiment) by load testing an HTTP service and validating its latency and error-related [service level objectives (SLOs)](../user-guide/topics/slos.md).
 
 ***
 
 ## 1. Install Iter8
 === "Brew"
+    Install the latest stable release of the Iter8 CLI using `brew` as follows.
+
     ```shell
     brew tap iter8-tools/iter8
     brew install iter8
     ```
+    
+    ??? note "Install a specific version"
+        You can install the latest stable release of the Iter8 CLI with specific major and minor version numbers. For example, the following command installs the latest stable release of the Iter8 CLI with major `0` and minor `8`.
+        ```shell
+        brew tap iter8-tools/iter8
+        brew install iter8@0.8
+        ```
```

Comment:
Do we need this here? This is a tutorial. No need to provide every detail about install.

Reply:
Specific version is hidden under the details twistie. I think it is useful to keep in "your first experiment". This is the only tutorial where it appears, and besides, `material mkdocs` makes it easy to keep it neat.